Master-OB-OpenAPI.json

```diff
@@ -11268,6 +11274,40 @@
             "x-ob-usage-tips": ""
           }
         ]
+      },
+      "AnswerRangeMin": {
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/TaxonomyElementInteger"
+          },
+          {
+            "type": "object",
+            "description": "Minimum range of acceptable input.",
+            "x-ob-item-type": "IntegerItemType",
```

Comment:
I think these should be TaxonomyElementNumber, to allow for integer and float values.

Reply:
We do needs to be a number to include Float values.

Reply:
Are we missing a NumberItemType? I'm finding integer, decimal but no number item type.

Reply:
Use DecimalItemType